paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -460,7 +460,10 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
             (formTypeForCode(selectionCode) as? FormType.MandateOnly)?.mandate
         } else {
             val savedSelection = selection as? PaymentSelection.Saved?
-            savedSelection?.mandateText(paymentMethodMetadata.merchantName, paymentMethodMetadata.hasIntentToSetup())
+            savedSelection?.mandateText(
```

Comment:
This is duplicated from above. Should we add an extension function that passes in paymentMethodMetadata?